WordPress/Classes/ViewRelated/Views/WPUploadStatusButton.m

```diff
@@ -3,53 +3,33 @@
 
 @implementation WPUploadStatusButton
 
-+ (id)buttonWithFrame:(CGRect)frame
+- (instancetype)initWithFrame:(CGRect)frame
 {
-    WPUploadStatusButton *button = [WPUploadStatusButton buttonWithType:UIButtonTypeSystem];
-    button.frame = frame;
-    button.titleLabel.textColor = [UIColor whiteColor];
-    button.titleLabel.font = [WPFontManager systemBoldFontOfSize:14.0];
-    button.backgroundColor = [UIColor clearColor];
-    button.titleLabel.adjustsFontSizeToFitWidth = NO;
-    button.autoresizingMask = UIViewAutoresizingFlexibleWidth;
-    [button setAccessibilityHint:NSLocalizedString(@"Tap to cancel uploading.", nil)];
-    [button setTitle:NSLocalizedString(@"Uploading...", @"\"Uploading...\" Status text") forState:UIControlStateNormal];
-    [button setAccessibilityHint:NSLocalizedString(@"Tap to select which blog to post to", @"This is the blog picker in the editor")];
-    button.titleLabel.numberOfLines = 1;
-    button.titleLabel.textAlignment = NSTextAlignmentNatural;
-    button.contentHorizontalAlignment = UIControlContentHorizontalAlignmentLeft;
-    button.titleLabel.lineBreakMode = NSLineBreakByTruncatingTail;
+    self = [super initWithFrame:frame];
+    if (!self) {
+        return nil;
+    }
+    self.frame = frame;
+    self.titleLabel.textColor = [UIColor whiteColor];
+    self.titleLabel.font = [WPFontManager systemBoldFontOfSize:14.0];
+    self.backgroundColor = [UIColor clearColor];
+    self.titleLabel.adjustsFontSizeToFitWidth = NO;
+    self.autoresizingMask = UIViewAutoresizingFlexibleWidth;
+    [self setAccessibilityHint:NSLocalizedString(@"Tap to cancel uploading.", @"This is a status indicator on the editor")];
+    [self setTitle:NSLocalizedString(@"Uploading", @"\"Uploading\" Status text") forState:UIControlStateNormal];
```

Comment:
I don't think the ... where adding anything because if the activity indicator in front of it. I think on the initial version we had the ... but no indicator so they made more sense.